### ISSUE TITLE
Velodrome Finance Optimism QA

### ISSUE THREAD

**Issue by SerenityFund** [messari/subgraphs#1120]
## Review Info

| Description | Info | 
| :-------- | :--- | 
| Subgraph Reviewed |  https://subgraphs.messari.io/subgraph?endpoint=l0c4t0r/laboratory5&tab=protocol
| Schema Version  |  1.3.0  | 
| Subgraph Version |  1.0.0  | 
| Methodology Version  |	1.0.0 |
| Evidence Spreadsheet |  https://docs.google.com/spreadsheets/d/1ykWUrYZAX6PLJC266t9yYMx7nkDR9tuG/edit?usp=sharing&ouid=104027705798814439445&rtpof=true&sd=true |

#### Daily Total Revenue and Cumulative Total Revenue

- We calculated the implied total revenue and the results are in line with the understanding that 0.02% of the daily volume is the daily fees or total revenue. 
- Data from CryptoFees' website suggests that subgraph data is accurate as it is mathematically in line with it. As for Dune Analytics data, there is a discrepancy above 100% against it in the observed days, however, data from Dune Analytics might be wrong as it is not the official Velodrome dashboard.
- The accumulated total revenue increased by a massive amount in 24th of September from 35 million to 269 million dollars. There is no other source to confirm this huge spike. 

|Subgraph's Daily Total Revenue |CryptoFees| Absolute Difference |% Difference |
| :---- | :--- | :---  | :---  |
|1,998.123 | 1,985.96 |12.16|0.61%|

|Subgraph's Daily Total Revenue |Dune Analytics| Absolute Difference |% Difference |
| :---- | :--- | :---  | :---  |
|1,998.123 | 929.8 |1,068.32|114.90%|

Note: Data above is from 28th of September

|Cumulative Total Revenue for Sept. 23 |Cumulative Total Revenue for Sept. 24| % Change |
| :---- | :--- | :---  | 
|34,906,084 | 268,787,586 |670.90%|

|Daily Total Revenue for Sept. 23 |Daily Total Revenue for Sept. 24| 
| :---- | :--- | 
|233,881,502 | 2,511 |

- Refer to the sheet 'Pool_Overview' 

### Editorials  

There are some errors we have noticed and are not necessarily data errors.

#### Pool Overview

- The pool overview section cannot be accessed and it is displaying some issues as it says 'Failed to fetch'
- Refer to the sheet 'Pool_Overview'   

#### Pool

- The subgraph query is not working no matter how many times the page is being refreshed.
- Refer to the sheet 'Pool_vault_schema' and 'Pool_snapshots'  

#### Events

- The subgraph is not showing any data and only shows black screen.
- Refer to the sheet 'Events'   

 Please refer to the attached worksheet for details.

**Comment by pietydespises** [comment]
I think we're OK to ignore the revenue discrepancies with Dune Analytics since its in line with CryptoFees. 

NO DATA RELATED ERRORS. ONLY VALIDATION DASHBOARD ERRORS:

-Pool Overview not loading showing on validation dashboard
-Pool tab not loading on validation dashboard
-Events not loading on validation dashboard

In fact at time of writing the validation dashboard is not accessible at all. @this-username-is-taken

**Comment by l0c4t0r** [comment]
Hi, sorry about the confusion. You must might want to look at: https://subgraphs.messari.io/subgraph?endpoint=l0c4t0r/laboratory3&tab=protocol

This fixes the bug regarding the revenue.
I had made a note of this in the PR but perhaps it wasn't clear.
I redeployed at laboratory3 as the sync was slow, so I wanted to let lab5 continue to sync despite the issue to spot any other problems.

Also I noticed the name of this issue refers to arbitrum but velodrome is on optimism.
